libevmasm/AssemblyItem.cpp

```diff
@@ -26,27 +26,35 @@ using namespace std;
 using namespace dev;
 using namespace dev::eth;
 
+static_assert(sizeof(size_t) == 8, "size_t must be at most 64-bits wide");
```

Comment:
I'll leave this in as `<= 8`.

We can address `== 8` in a separate PR / discussion, because that affects a lot more things. It is possible to switch to 64 bit size_t in emscripten I'd assume.